tests_pytest/keras_tests/_keras_base_tests/base_keras_mct_bypass.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2025 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import keras
+import numpy as np
+from tests_pytest._fw_tests_common_base.base_mct_bypass_test import BaseMCTBypassTest
+
+
+class BaseKerasMCTBypassTest(BaseMCTBypassTest):
+
+
```

Comment:
Can you add a short description of what the test does?
This test class is designed to verify that a PyTorch model, when processed through the MCT with a bypass enabled, retains its original architecture and parameters.